README.md

```diff
@@ -67,6 +67,35 @@ person.email # => "me@example.org"
 person.age   # => raises NoMethodError because `:age` wasn't defined as attribute.
 ```
 
+#### Blank Values
+
+The framework will treat as valid any blank attributes, __without__ `presence`, for both `format` and `size` predicates.
+
+```ruby
+require 'lotus/validations'
+
+class Person
+  include Lotus::Validations
+
+  attribute :name,    type: String, size: 5..45
+  attribute :email,   type: String, size: 20..80, format: /@/
+  attribute :skills,  type: Array,  size: 1..3
+  attribute :keys,    type: Hash,   size: 1..3
+end
+
+Person.new.valid?                             # < true
+Person.new(name: '').valid?                   # < true
+Person.new(email: '@').valid?                 # < true
```

Comment:
You're right. By a mistaken, I've thought one thing and did another totally different. Good catch! Thanks :+1: